tfjs-backend-webgpu/src/kernels/resize_bilinear_webgpu.ts

```diff
@@ -41,7 +43,15 @@ export class ResizeBilinearProgram implements WebGPUProgram {
         this.dispatchLayout, this.outputShape, this.workGroupSize);
 
     this.alignCorners = alignCorners;
-    this.shaderKey = `resizeBilinear_${alignCorners}_${
+    this.halfPixelCenters = halfPixelCenters;
+    if (halfPixelCenters) {
+      this.sourceFracIndexRC =
+          `(vec2(rc) + vec2(0.5)) * effectiveInputOverOutputRatioRC` +
+          ` - vec2(0.5)`;
+    } else {
+      this.sourceFracIndexRC = `vec2(rc) * effectiveInputOverOutputRatioRC`;
+    }
```

Comment:
Please move above code snippet to the corresponding `getUserCode[wgsl]` function. I believe the format is different for glsl and wgsl. Otherwise, it may have compiling error for the wgsl path.

Reply:
Thanks for pointing this, fixed PTAL